packages/web-extension-v2/package.json

```diff
@@ -30,15 +30,16 @@
     "@carbon/type": "^10.33.0",
     "carbon-components": "^10.41.0",
     "carbon-components-react": "^7.41.0",
+    "carbon-devtools": "^2.7.7",
```

Comment:
Is the v2 version dependent on the v3 Version?

Reply:
Yes, it is. https://github.com/carbon-design-system/devtools/blob/base/packages/web-extension-v2/webpack.common.js#L17-L31